backend/scaife_stack_atlas/hooks.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path  # noqa: F401
+
+from scaife_viewer.atlas.hooks import DefaultHookSet
+
+
+class ATLASHookSet(DefaultHookSet):
+    def get_prepared_tokens(self, version_urn):
+        # NOTE: Sample override for The life of Omar ben Saeed below
+        if version_urn == "urn:cts:arabicLit:amedsaid1831.dw042.perseus-ara1:":
+            from scaife_viewer.atlas.tokenizers import get_tokens_from_csv
+            csv_path = Path("data/token-overrides/amedsaid1831.dw042.perseus-ara1.csv")
```

Comment:
@jchill-git This my "draft" placement for these files; I think we will likely establish a more formal convention in the future.

You can replace my stub file with your actual tokens once you've got them.